src/routes/indexable-indexation-route.php

```diff
@@ -243,6 +243,7 @@ public function prepare() {
 	 */
 	public function complete() {
 		$this->options_helper->set( 'indexation_started', 0 );
+		$this->options_helper->set( 'indexables_indexation_reason', 0 );
```

Comment:
You made it a string value and the `has_indexation_reason` checks on an empty string.
```suggestion
		$this->options_helper->set( 'indexables_indexation_reason', '' );
```